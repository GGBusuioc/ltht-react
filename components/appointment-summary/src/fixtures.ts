import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
