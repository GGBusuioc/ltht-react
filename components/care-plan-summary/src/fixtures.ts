import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
