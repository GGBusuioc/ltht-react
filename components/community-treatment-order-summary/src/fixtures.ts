import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
