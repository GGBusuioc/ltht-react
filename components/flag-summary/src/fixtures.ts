import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
