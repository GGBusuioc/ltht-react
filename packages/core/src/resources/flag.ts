import { CodeableConcept } from './codeable-concept'
import { Identifier } from './identifier'
import { ResourceReference } from './resource-reference'
import { Resource } from './resource'
import { Period } from './period'

enum StatusCode {
  ACTIVE,
  INACTIVE,
  ENTERED_IN_ERROR,
}

interface Flag extends Resource {
  readonly identifier?: Identifier[] | null | undefined
  readonly status?: StatusCode | null | undefined
  readonly category?: CodeableConcept | null | undefined
  readonly code?: CodeableConcept | null | undefined
  readonly period?: Period | null | undefined
  readonly author?: ResourceReference | null | undefined
}

export { Flag, StatusCode }
