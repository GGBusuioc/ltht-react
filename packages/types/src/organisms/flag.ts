import { CodeableConcept } from '../molecules/codeable-concept'
import { Identifier } from '../atoms/identifier'
import { ResourceReference } from '../molecules/resource-reference'
import { Resource } from './resource'
import { Period } from '../atoms/period'

enum StatusCode {
  ACTIVE,
  INACTIVE,
  ENTERED_IN_ERROR,
}

interface Flag extends Resource {
  readonly identifier?: Identifier[] | null | undefined
  readonly status?: StatusCode | null | undefined
  readonly category?: CodeableConcept | null | undefined
  readonly code?: CodeableConcept | null | undefined
  readonly period?: Period | null | undefined
  readonly author?: ResourceReference | null | undefined
}

export { Flag, StatusCode }
