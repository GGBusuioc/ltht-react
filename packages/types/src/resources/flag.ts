import { CodeableConcept } from './codeable-concept'
import { Identifier } from './identifier'
import { ResourceReference } from './resource-reference'
import { Resource } from './resource'
import { Period } from './period'

enum StatusCode {
  ACTIVE,
  INACTIVE,
  ENTERED_IN_ERROR,
}

interface Flag extends Resource {
  readonly identifier?: Identifier[] | undefined
  readonly status?: StatusCode | undefined
  readonly category?: CodeableConcept | undefined
  readonly code?: CodeableConcept | undefined
  readonly period?: Period | undefined
  readonly author?: ResourceReference | undefined
}

export { Flag, StatusCode }
