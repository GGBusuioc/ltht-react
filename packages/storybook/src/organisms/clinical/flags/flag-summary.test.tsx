import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import FlagSummary from '@ltht-react/flag-summary'
import { FlagOne, FlagTwo } from './flag.fixtures'

describe('FlagSummary', () => {
  const flags = [FlagOne, FlagTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<FlagSummary flags={flags} />, div)
  })
  it('matches snapshot', () => {
    expect(mount(<FlagSummary flags={flags} />)).toMatchSnapshot('wrapper mount')
  })
})
