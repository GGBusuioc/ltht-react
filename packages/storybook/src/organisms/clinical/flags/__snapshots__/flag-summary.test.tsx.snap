// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlagSummary matches snapshot: wrapper mount 1`] = `
.emotion-16 {
  color: #231f20;
  font-family: 'SF Pro Text','SF Pro Icons','Helvetica Neue',Helvetica,Arial,sans-serif;
  font-size: 0.8rem;
  background: #FFFFFF;
  color: #231f20;
  margin-bottom: 0.5rem;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(102,102,102,0.1), 0px 1px 1px 0px rgba(102,102,102,0.15), 0px 1px 3px 0px rgba(102,102,102,.6);
  margin: 0 5px 10px 0;
  -webkit-font-smoothing: antialiased;
}

.emotion-16 *,
.emotion-16 ::before,
.emotion-16 ::after {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

.emotion-16 * {
  margin: 0;
  padding: 0;
}

.emotion-16 h3 {
  font-size: 1rem;
}

@media only screen and (min-width:768px) {
  .emotion-16 {
    font-size: 0.9rem;
  }

  .emotion-16 h3 {
    font-size: 1.1rem;
  }
}

@media only screen and (min-width:1024px) {
  .emotion-16 {
    font-size: 1rem;
  }

  .emotion-16 h3 {
    font-size: 1.2rem;
  }
}

@media only screen and (min-width:1200px) {
  .emotion-16 {
    font-size: 1.1rem;
  }

  .emotion-16 h3 {
    font-size: 1.3rem;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0.5rem;
}

.emotion-2:hover {
  cursor: pointer;
}

.emotion-0 {
  color: #768692;
}

.emotion-15 {
  list-style: none;
  padding-left: 0.5rem;
}

.emotion-15 li {
  border-top: 1px solid #b0b0b0;
  padding: 0.5rem 0.5rem 0.5rem 0;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-5 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-3 {
  color: #231f20;
  text-align: left;
}

.emotion-4 {
  color: #768692;
  text-align: left;
}

.emotion-6 {
  color: #231f20;
  text-align: right;
}

.emotion-7 {
  color: #768692;
  text-align: right;
}

<FlagSummary
  flags={
    Array [
      Object {
        "author": Object {
          "display": "Smith, John (Dr)",
          "identifier": undefined,
          "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
          "typeName": "Practitioner",
        },
        "category": Object {
          "coding": Array [
            Object {
              "code": "66706",
              "display": "Location",
            },
            Object {
              "code": "66707",
              "display": "Demographics",
            },
          ],
        },
        "code": Object {
          "coding": Array [
            Object {
              "code": "109006",
              "display": "Lives in Known High Risk Area",
            },
          ],
        },
        "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "start": Object {
            "kind": "YEAR",
            "value": "2013-01-01T00:00:00+00:00",
          },
        },
        "status": "ENTERED_IN_ERROR",
        "text": Object {
          "div": "Lorem ipsum dolor sit amet",
          "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
        },
      },
      Object {
        "author": Object {
          "display": "John Smith",
          "identifier": undefined,
          "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
          "typeName": "Practitioner",
        },
        "category": Object {
          "coding": Array [
            Object {
              "code": "66706",
              "display": "Violence",
            },
          ],
        },
        "code": Object {
          "coding": Array [
            Object {
              "code": "56706",
              "display": "High Risk of Violence",
            },
          ],
        },
        "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "end": Object {
            "kind": "DATE",
            "value": "2014-02-02T00:00:00+00:00",
          },
          "start": Object {
            "kind": "DATE",
            "value": "2013-01-01T00:00:00+00:00",
          },
        },
        "status": "INACTIVE",
      },
    ]
  }
  title="Alerts"
>
  <Card
    noData={false}
  >
    <CardProvider
      childrenCount={2}
      collapsible={true}
      noData={false}
    >
      <div
        className="emotion-16"
      >
        <CardHeader>
          <div
            className="emotion-2"
            onClick={[Function]}
            title="Click to collapse"
          >
            <div>
              <h3>
                Alerts
              </h3>
            </div>
            <div>
              <ChevronCircleIcon
                direction="down"
                size="medium"
              >
                <FontAwesomeIcon
                  border={false}
                  className="emotion-0"
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        512,
                        512,
                        Array [],
                        "f13a",
                        "M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z",
                      ],
                      "iconName": "chevron-circle-down",
                      "prefix": "fas",
                    }
                  }
                  inverse={false}
                  listItem={false}
                  mask={null}
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size="sm"
                  spin={false}
                  swapOpacity={false}
                  symbol={false}
                  title=""
                  transform={null}
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-chevron-circle-down fa-w-16 fa-sm emotion-0"
                    data-icon="chevron-circle-down"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                </FontAwesomeIcon>
              </ChevronCircleIcon>
            </div>
          </div>
        </CardHeader>
        <CardList
          clickable={false}
        >
          <ul
            className="emotion-15"
          >
            <ListItem
              key="580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard"
            >
              <li>
                <FlagSummaryItem
                  flag={
                    Object {
                      "author": Object {
                        "display": "Smith, John (Dr)",
                        "identifier": undefined,
                        "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                        "typeName": "Practitioner",
                      },
                      "category": Object {
                        "coding": Array [
                          Object {
                            "code": "66706",
                            "display": "Location",
                          },
                          Object {
                            "code": "66707",
                            "display": "Demographics",
                          },
                        ],
                      },
                      "code": Object {
                        "coding": Array [
                          Object {
                            "code": "109006",
                            "display": "Lives in Known High Risk Area",
                          },
                        ],
                      },
                      "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                        "isRedacted": true,
                        "requestedWhen": "",
                      },
                      "period": Object {
                        "start": Object {
                          "kind": "YEAR",
                          "value": "2013-01-01T00:00:00+00:00",
                        },
                      },
                      "status": "ENTERED_IN_ERROR",
                      "text": Object {
                        "div": "Lorem ipsum dolor sit amet",
                        "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
                      },
                    }
                  }
                >
                  <div
                    className="emotion-8"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <FlagTitle
                        flag={
                          Object {
                            "author": Object {
                              "display": "Smith, John (Dr)",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Location",
                                },
                                Object {
                                  "code": "66707",
                                  "display": "Demographics",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "109006",
                                  "display": "Lives in Known High Risk Area",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "start": Object {
                                "kind": "YEAR",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ENTERED_IN_ERROR",
                            "text": Object {
                              "div": "Lorem ipsum dolor sit amet",
                              "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          Lives in Known High Risk Area
                        </div>
                      </FlagTitle>
                      <FlagDescription
                        flag={
                          Object {
                            "author": Object {
                              "display": "Smith, John (Dr)",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Location",
                                },
                                Object {
                                  "code": "66707",
                                  "display": "Demographics",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "109006",
                                  "display": "Lives in Known High Risk Area",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "start": Object {
                                "kind": "YEAR",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ENTERED_IN_ERROR",
                            "text": Object {
                              "div": "Lorem ipsum dolor sit amet",
                              "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          109006 - Location, Demographics
                        </div>
                      </FlagDescription>
                    </div>
                    <div>
                      <FlagDate
                        flag={
                          Object {
                            "author": Object {
                              "display": "Smith, John (Dr)",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Location",
                                },
                                Object {
                                  "code": "66707",
                                  "display": "Demographics",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "109006",
                                  "display": "Lives in Known High Risk Area",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "start": Object {
                                "kind": "YEAR",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ENTERED_IN_ERROR",
                            "text": Object {
                              "div": "Lorem ipsum dolor sit amet",
                              "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          2013 to Present
                        </div>
                      </FlagDate>
                      <FlagStatus
                        flag={
                          Object {
                            "author": Object {
                              "display": "Smith, John (Dr)",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Location",
                                },
                                Object {
                                  "code": "66707",
                                  "display": "Demographics",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "109006",
                                  "display": "Lives in Known High Risk Area",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe5|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "start": Object {
                                "kind": "YEAR",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ENTERED_IN_ERROR",
                            "text": Object {
                              "div": "Lorem ipsum dolor sit amet",
                              "text": "Lorem ipsum dolor sit amet, falli nulla oportere vel eu. Ea meis moderatius scripserit his, ei decore ceteros has. Quo at apeirian facilisi consectetuer. At putant animal persius his, inani maiestatis moderatius vis no",
                            },
                          }
                        }
                      >
                        <div
                          className="emotion-7"
                        >
                          Entered In Error
                        </div>
                      </FlagStatus>
                    </div>
                  </div>
                </FlagSummaryItem>
              </li>
            </ListItem>
            <ListItem
              key="580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard"
            >
              <li>
                <FlagSummaryItem
                  flag={
                    Object {
                      "author": Object {
                        "display": "John Smith",
                        "identifier": undefined,
                        "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                        "typeName": "Practitioner",
                      },
                      "category": Object {
                        "coding": Array [
                          Object {
                            "code": "66706",
                            "display": "Violence",
                          },
                        ],
                      },
                      "code": Object {
                        "coding": Array [
                          Object {
                            "code": "56706",
                            "display": "High Risk of Violence",
                          },
                        ],
                      },
                      "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                        "isRedacted": true,
                        "requestedWhen": "",
                      },
                      "period": Object {
                        "end": Object {
                          "kind": "DATE",
                          "value": "2014-02-02T00:00:00+00:00",
                        },
                        "start": Object {
                          "kind": "DATE",
                          "value": "2013-01-01T00:00:00+00:00",
                        },
                      },
                      "status": "INACTIVE",
                    }
                  }
                >
                  <div
                    className="emotion-8"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <FlagTitle
                        flag={
                          Object {
                            "author": Object {
                              "display": "John Smith",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Violence",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "56706",
                                  "display": "High Risk of Violence",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "INACTIVE",
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          High Risk of Violence
                        </div>
                      </FlagTitle>
                      <FlagDescription
                        flag={
                          Object {
                            "author": Object {
                              "display": "John Smith",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Violence",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "56706",
                                  "display": "High Risk of Violence",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "INACTIVE",
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          56706 - Violence
                        </div>
                      </FlagDescription>
                    </div>
                    <div>
                      <FlagDate
                        flag={
                          Object {
                            "author": Object {
                              "display": "John Smith",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Violence",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "56706",
                                  "display": "High Risk of Violence",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "INACTIVE",
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          01-Jan-2013 to 02-Feb-2014
                        </div>
                      </FlagDate>
                      <FlagStatus
                        flag={
                          Object {
                            "author": Object {
                              "display": "John Smith",
                              "identifier": undefined,
                              "reference": "https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser",
                              "typeName": "Practitioner",
                            },
                            "category": Object {
                              "coding": Array [
                                Object {
                                  "code": "66706",
                                  "display": "Violence",
                                },
                              ],
                            },
                            "code": Object {
                              "coding": Array [
                                Object {
                                  "code": "56706",
                                  "display": "High Risk of Violence",
                                },
                              ],
                            },
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                              "isRedacted": true,
                              "requestedWhen": "",
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "INACTIVE",
                          }
                        }
                      >
                        <div
                          className="emotion-7"
                        >
                          Inactive
                        </div>
                      </FlagStatus>
                    </div>
                  </div>
                </FlagSummaryItem>
              </li>
            </ListItem>
          </ul>
        </CardList>
      </div>
    </CardProvider>
  </Card>
</FlagSummary>
`;
